manifests/params.pp

```diff
@@ -30,6 +30,12 @@
   ]
   $purge_plugins = false
 
+  if versioncmp(pick($facts['jenkins_version'], '0'), '2.313') < 0 {
```

Comment:
Doesn't this still default to forking when the fact is missing? `'0' < '2.313'`, right? Could you add a test for this?

Reply:
like this it's ok for you ?